config/finders/all_content.yml

```diff
@@ -31,9 +25,7 @@ details:
     name: Manual
     preposition: in manual
     short_name: in
-    type: hidden
-    show_option_select_filter: false
-    allowed_values: []
+    type: hidden_clearable
```

Comment:
Is this the only change we need?  I think the other changes look more like a frontend schema - for example we don't need the supertypes, and we definitely do need the routes.

In other news, I noticed yesterday that the finderpresenter tests aren't running properly, so they're not picking up schema violations where they should here: https://github.com/alphagov/rummager/blob/master/spec/unit/content_item_publisher/finder_presenter_spec.rb#L30.  I was going to have a poke to fix them.

Reply:
Apologies, act in haste, repent at leisure, fixed now

Reply:
We've all done it 🤦‍♂️